hypersistence-utils-hibernate-62/src/test/java/io/hypersistence/utils/hibernate/id/AbstractBatchSequenceGeneratorTest.java

```diff
@@ -93,15 +93,10 @@ protected long getSelectCount(QueryCount queryCount) {
     public static class Post {
 
         @Id
-        @GenericGenerator(
-            name = "post_sequence",
-            strategy = "io.hypersistence.utils.hibernate.id.BatchSequenceGenerator",
-            parameters = {
-                    @Parameter(name = "sequence", value = "SEQ_PARENT_ID"),
-                    @Parameter(name = "fetch_size", value = "" + BATCH_SIZE)
-            }
+        @BatchSequence(
```

Comment:
I didn't manage to run the tests. `io.hypersistence.utils.spring.repo.base.SpringDataJPABaseTest` went to log level debug and they would never finish.